src/pages/Homepage.tsx

```diff
@@ -0,0 +1,32 @@
+import { CategoryContainer, QuizCard, SiteIntroduction } from "../components";
+import { quizCategoryData, quizData } from "toolkit/data/quizData";
+
+export const Homepage = () => {
+  return (
+    <>
+      <SiteIntroduction />
+      <main className="m-auto">
+        <section id="categories" className="section centered-text grid-ctr w-95p m-t-3 m-b-5">
```

Comment:
why you have used `id` here?

Reply:
Added this for in page scrolling effect.